index_settings_test.go

```diff
@@ -1222,6 +1320,15 @@ func TestIndex_UpdateSettings(t *testing.T) {
 					SortableAttributes: []string{
 						"title",
 					},
+					TypoTolerance: &TypoTolerance{
```

Comment:
Do you need to check in every assertion if the object is equal to another object? What do you think about making the assertions with the part you need?

I saw a lot of changed tests with `TypoTolerance` stuff which did not concern them!

Reply:
I don't understand what you mean. Can you explain a little more?

Reply:
Yes, but I think the answer will be "the tests are autogenerated" haha.

What I mean is, if in the test "update typo tolerance using settings" we just update one field, we don't need to assert every other field, just we want to be changed.